reference.md

```diff
@@ -0,0 +1,79 @@
+Cheat sheet of functions used in the lessons
+
+
+**Lesson 1**
```

Comment:
could you please add the title of the lessons here and below (e.g. **Lesson 1: Introduction to R**)?